test/DynamoCoreTests/PackageManager/PackageTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using NUnit.Framework;
+
+namespace Dynamo.PackageManager
+{
+    
+    class PackageTests
+    {
+        private readonly AssemblyName assemName =
+            new AssemblyName("Rhynamo, Version=2013.3.6.0, Culture=neutral, PublicKeyToken=null");
```

Comment:
I would change this name out for something generic just in case @archinate ever decides to trademark the name.